### DISCUSSION TITLE
可以提供一个简易的直接根据地址在应用内更新的插件么？

### DISCUSSION THREAD

**Discussion by sea-forever-zh** [site-chenwei/code-update#2] [Q&A] [answered]
我想在ionic6+angular14+capacitor4的环境中，在应用内更新。
但是我在npm，github上没有找到合适的包。
如果能提供，万分感谢🦀🦀

**Comment by site-chenwei** [comment, marked as answer]
你具体说一下你的需求是啥呢？用来做类似于离线包的功能吗？

**Reply by sea-forever-zh** [reply]
嗯，就是直接通过自己服务器上的apk地址更新。
我已找到一个可用的了。kolbasa/cordova-plugin-apkupdater
谢谢关注
